src/web/app/components/comment-box/comment-edit-form/comment-edit-form.component.html

```diff
@@ -47,10 +47,26 @@
                        placeholderText="Enter your comment here" [isDisabled]="isDisabled"></tm-rich-text-editor>
 </div>
 
-<button type="button" class="btn btn-success btn-sm float-end btn-margin-left"
-        (click)="triggerSaveCommentEvent()" *ngIf="!shouldHideSavingButton" [disabled]="isDisabled"><i class="fas fa-save"></i>
-  {{ mode === CommentRowMode.EDIT ? 'Save' : 'Add' }}</button>
+<div class="d-flex justify-content-end">
+  <button
+    type="button"
+    class="btn btn-light btn-sm me-2"
+    (click)="triggerCloseCommentBoxEvent()"
+    *ngIf="!shouldHideClosingButton"
+    [disabled]="isDisabled"
+    aria-label="Cancel editing">
+    <i class="fas fa-times"></i>
+    Cancel
+  </button>
 
-<button type="button" class="btn btn-light btn-sm float-end"
-        (click)="triggerCloseCommentBoxEvent()" *ngIf="!shouldHideClosingButton" [disabled]="isDisabled"><i class="fas fa-times"></i>
-  Cancel</button>
+  <button
+    type="button"
+    class="btn btn-success btn-sm"
+    (click)="triggerSaveCommentEvent()"
+    *ngIf="!shouldHideSavingButton"
+    [disabled]="isDisabled"
+    aria-label="Save comment">
+    <i class="fas fa-save"></i>
+    {{ mode === CommentRowMode.EDIT ? 'Save' : 'Add' }}
+  </button>
+</div>
```

Comment:
I think the original intention was to swap only the order for the screen reader and not the actual positions of the buttons.

Cant get my screen reader to work to test but if im not wrong this I think this still reads from right to left, so it will read `Cancel` then `Add`? 
![image](https://github.com/user-attachments/assets/b9bf10a3-b6e4-4b9e-b075-71861750f756)

Also small nit, if you manage to find a solution to this, the gap between the buttons is also missing here

Reply:
@dishenggg For the screen reader to go in order, changes would be required in the DOM as it follows the DOM model. Previously, the order of the buttons does not match the layout (left to right). I have changed it to match the expected layout while not changing the order of the buttons in the UI. I have tried it out and the expected output has been achieved with these changes. 
Please feel free to let me know if you require any other clarifications!

Reply:
On that note, I realised that you might not have been on the correct branch/page, 
The PR fixes the issue on the Instructor's student record page, which you should be seeing something similar to this: 
![Screenshot 2024-08-01 at 12 32 09 AM](https://github.com/user-attachments/assets/78a14ce4-8314-4615-b91c-69d009c20b36)

Reply:
I think you are right, I might have connected to the Backend url with an older front-end build. Thanks for the PR!